Doc/library/math.rst

```diff
@@ -27,36 +27,39 @@ noted otherwise, all return values are floats.
 
 
 ====================================================  ============================================
-**Number-theoretic and representation functions**
+**Number-theoretic functions**
 --------------------------------------------------------------------------------------------------
-:func:`ceil(x) <ceil>`                                Ceiling of *x*, the smallest integer greater than or equal to *x*
 :func:`comb(n, k) <comb>`                             Number of ways to choose *k* items from *n* items without repetition and without order
-:func:`copysign(x, y) <copysign>`                     Magnitude (absolute value) of *x* with the sign of *y*
-:func:`fabs(x) <fabs>`                                Absolute value of *x*
 :func:`factorial(n) <factorial>`                      *n* factorial
+:func:`gcd(*integers) <gcd>`                          Greatest common divisor of the integer arguments
+:func:`isqrt(n) <isqrt>`                              Integer square root of a nonnegative integer *n*
+:func:`lcm(*integers) <lcm>`                          Least common multiple of the integer arguments
+:func:`perm(n, k) <perm>`                             Number of ways to choose *k* items from *n* items without repetition and with order
+
+**Floating point arithmetic**
```

Comment:
I think you should sync table names with actual section headers.

Reply:
Of course! I forgot to update the section header, it's done now.